examples/bower/index.html

```diff
@@ -5,9 +5,18 @@
         accessToken: "ACCESS_TOKEN",
         captureUncaught: true,
         payload: {
-          environment: "test"
+          environment: "test",
+          person: {
+            id: 456,
+            username: "foo",
+            email: "foo@example.com"
+          }
         }
       };
+      
+      function breakme() {
+        Rollbar.error("This will be reported to Rollbar");
```

Comment:
Would be nice to show an actual Error object being reported here (so that there is a stack trace). e.g.

```js
try {
  var foo = null;
  foo()
} catch (e) {
  Rollbar.error(e);
}
```